_posts/2021-10-04-SE2022-RSE-Track-Call-for-contributions-de.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "rSE22 Call for Contributions"
+layout: post
+author: "Stephan Janosch, rSE22 Programmkommittee"
+menulang: de
+---
+
+# rSE22 - Call for Contributions
+Im Juni 2019 trafen sich bei der [deRSE19](https://de-rse.org/en/conf2019/) in Potsdam Forschungssoftwareentwickler*innen aus ganz Deutschland zum ersten Mal zu einer zentralen Konferenz. Ideen und Pläne für Folgekonferenzen in 2020 und 2021  machte leider die Pandemie einen Strich durch die Rechnung. Daher bündelt de-RSE e.V. jetzt seine Kräfte mit Kollegen aus dem Software Engineering und organisiert für die [SE 2022](https://www.se-2022.de)   der Gesellschaft für Informatik (GI)  im Februar 2022 einen separaten Track zum Thema Forschungssoftware. 
```

Comment:
```suggestion
Im Juni 2019 trafen sich bei der [deRSE19](https://de-rse.org/en/conf2019/) in Potsdam Forschungssoftwareentwickler*innen aus ganz Deutschland zum ersten Mal zu einer zentralen Konferenz. Ideen und Plänen für Folgekonferenzen in 2020 und 2021 machte leider die Pandemie einen Strich durch die Rechnung. Daher bündelt de-RSE e.V. jetzt seine Kräfte mit Kolleg*innen aus dem Software Engineering und organisiert für die [SE 2022](https://www.se-2022.de) der Gesellschaft für Informatik (GI) im Februar 2022 einen separaten Track zum Thema Forschungssoftware. 
```

Reply:
Better check rendering of gender `*` vs. `\*` locally, I'm not sure if the converter does this reliably.